packages/serverless-nextjs-plugin/classes/NextPage.js

```diff
@@ -6,6 +6,7 @@ const PluginBuildDir = require("./PluginBuildDir");
 
 class NextPage {
   constructor(pagePath, { serverlessFunctionOverrides, routes } = {}) {
+    this.isApi = pagePath.indexOf("sls-next-build/api") === 0;
```

Comment:
```suggestion
    this.isApi = pagePath.indexOf(path.join("sls-next-build", "api")) === 0;
```

Reply:
I believe this is breaking the int. tests because pagePath in Windows would be `sls-next-build\\api`

Reply:
Cheers, easy to forget about windows as a linux developer.